src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/AbstractServerTask.groovy

```diff
@@ -304,9 +304,6 @@ abstract class AbstractServerTask extends AbstractTask {
     }
 
     protected String getBaseName(Task task) {
-        if (springBootVersion?.startsWith('1')) {
```

Comment:
Not much point in having a method any more when it just returns task.baseName. Is it used many places? If not, I would just remove this method and fix those references.

Reply:
Removed